extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -186,6 +187,27 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
         'advancementMode': AdvancementMode.MANUAL_ADVANCE,
       });
     }
+
+    if (
+      event.target.classList.contains(
+        'i-amphtml-story-subscriptions-google-button'
+      )
+    ) {
+      const observer = new MutationObserver((mutationsList) => {
+        mutationsList.forEach((mutation) => {
+          mutation.addedNodes.forEach((addedNode) => {
+            if (addedNode.tagName === 'SWG-POPUP-BACKGROUND') {
+              setImportantStyles(addedNode, {'pointer-events': 'all'});
+              observer.disconnect();
+            }
+          });
+        });
+      });
+      observer.observe(this.win.document.body, {
+        subtree: false,
+        childList: true,
+      });
+    }
```

Comment:
Why not move this to initializeListeners_?
If we do that we might need to remove observer.disconnect.